README.md

```diff
@@ -54,8 +54,8 @@ Dapr is currently under community development with preview releases. The master
 Find SDK-specific samples in the links below:
 
 - **[.NET SDK](https://github.com/dapr/dotnet-sdk)**
-  - **[Getting Started with ASP.NET Core](https://github.com/dapr/dotnet-sdk/tree/master/samples/AspNetCore)** - Samples for developing ASP.NET applications using the Dapr .NET SDK
-  - **[Getting Started with .NET Actors](https://github.com/dapr/dotnet-sdk/blob/master/docs/get-started-dapr-actor.md)** - Tutorial for developing actor applications using the Dapr .NET SDK including  **[actor samples](https://github.com/dapr/dotnet-sdk/tree/master/samples/Actor)**
+  - **[Getting Started with ASP.NET Core](https://github.com/dapr/dotnet-sdk/tree/master/examples/AspNetCore)** - Samples for developing ASP.NET applications using the Dapr .NET SDK
+  - **[Getting Started with .NET Actors](https://github.com/dapr/dotnet-sdk/blob/master/daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/dotnet-actor-howto.md)** - Tutorial for developing actor applications using the Dapr .NET SDK including  **[actor samples](https://github.com/dapr/dotnet-sdk/tree/master/examples/Actor)**
```

Comment:
Not sure about this one. The old file seems to be completely removed.

Reply:
So we're going to be adding this page to our rendered docs via this PR:
https://github.com/dapr/docs/pull/1149

Once that gets merged and the new docs rendered, I'll get you the right link for the dotnet actor howto.

Reply:
```suggestion
  - **[Getting Started with .NET Actors](https://docs.dapr.io/developing-applications/sdks/dotnet/dotnet-actors/dotnet-actor-howto/)** - Tutorial for developing actor applications using the Dapr .NET SDK including **[actor samples](https://github.com/dapr/dotnet-sdk/tree/master/examples/Actor)**
```